_output.yml

```diff
@@ -1,2 +1,19 @@
 bookdown::gitbook:
   css: https://ropensci.github.io/dev_guide/style.css
+  config:
+    sharing:
+      github: yes
+      facebook: false
+    toc:
+      collapse: subsection
+      before: |
+        <li><a href="./">The drake R Package User Manual</a></li>
+      after: |
+        <li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>
+    download: [pdf]
```

Comment:
Since we are including a pdf download link, could we add [front matter to generate the manual as a pdf](https://github.com/ropensci/dev_guide/blob/df87098955698b68066c4d7efe1cb5f7c93c1910/_output.yml#L20-L22)?

Reply:
Yes! My only worry is that if I edit preamble.tex in dev_guide, it wouldn't be updated here :thinking:

Reply:
Is it possible to link remotely to `preamble.tex` like I did with the stylesheet?

Reply:
...or would that not solve the problem?

Reply:
I'm not sure one can link remotely the .tex, but maybe?

Reply:
If it were possible, I would probably prefer to use https://ropensci.github.io/dev_guide/preamble.tex, but the `.tex` file is not hosted there. On the other hand, the preamble file itself only contains a single line, so I do not think I would mind a local copy in `drake-manual`.

Reply:
I added the lines and preamble.tex :-) I'm not especially proud of how the PDF looks like but I think it at least looks better than the default PDF :speak_no_evil:

Reply:
Thanks!